docs/architecture/Makefile

```diff
@@ -0,0 +1,18 @@
+PANDOC = pandoc
```

Comment:
Should this be `PANDOC ?= pandoc`?

Reply:
Should it? It's plain `=` in Mbed Crypto. `=` has the advantage of maximum portability, but we assume GNU make in our main makefile anyway, and `?=` exists in BSD make.

Reply:
Does BSD make have `$@ $<`?

Reply:
Yes, these are traditional make features.

Reply:
Didn't know that. I always had trouble using them with BSD make.